### PR TITLE
add links to provider auth docs and upgrade AWS to 1.1.0

### LINKED ISSUE #533: Add links to provider auth in quickstarts
### What's Missing?
The official providers are getting new auth docs
* [aws](https://docs.upbound.io/providers/provider-aws/authentication/)
* [gcp](https://docs.upbound.io/providers/provider-gcp/authentication/)

When Azure is published as well we should add links to these in each of the quickstart guides.

### PR DESCRIPTION
**Pull request by plumbis**
Adds links to the docs.upbound.io authentication docs for each provider.

For v1.15 and master this also upgrades provider AWS to v1..1.0

Resolves #533

Signed-off-by: Pete Lumbis <pete@upbound.io>

### CODE REVIEW

File: content/master/getting-started/provider-aws.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ kind: Provider
 metadata:
   name: provider-aws-s3
 spec:
-  package: xpkg.upbound.io/upbound/provider-aws-s3:v0.47.0
+  package: xpkg.upbound.io/upbound/provider-aws-s3:v0.1.0
```
Comment (jbw976) [inline content/master/getting-started/provider-aws.md:40]: I would expect this one to be `v1.1.0` like the others

File: content/v1.15/getting-started/provider-aws.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ kind: Provider
 metadata:
   name: provider-aws-s3
 spec:
-  package: xpkg.upbound.io/upbound/provider-aws-s3:v0.47.0
+  package: xpkg.upbound.io/upbound/provider-aws-s3:v0.1.0
```
Comment (jbw976) [inline content/v1.15/getting-started/provider-aws.md:40]: same issues here, should probably be `v1.1.0`

**Review by jbw976 — APPROVED** [review]
Small potential typo for a version number, but otherwise looks good and I tested the new authentication call outs and the links seem to point to the right place and work OK!

**Comment by plumbis** [comment]
Great catch! Thank you. Fixed and merged.

**Pull request merged**